config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  root 'root#index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
```

Comment:
- [ ] I noticed you do not have a route for the api, there is no way it would render. Kindly define a route for it.